qcodes_contrib_drivers/drivers/CopperMountain/M5180.py

```diff
@@ -0,0 +1,572 @@
+        terminator (str, optional, default "\n"): Terminator character of
+            the string reply.
+        timeout (int, optional, default 100000): VISA timeout is set purposely
+            to a long time to allow long spectrum measurement.
+        """
+
+        super().__init__(name       = name,
+                         address    = address,
+                         terminator = terminator,
+                         timeout    = timeout,
+                         **kwargs)
+
+        self.add_function('reset', call_cmd='*RST')
+
+        self.add_parameter(name='output',
+                           label='Output',
+                           get_parser=str,
+                           get_cmd='OUTP:STAT?',
+                           set_cmd='OUTP:STAT {}',
+                           val_mapping=create_on_off_val_mapping(on_val='1',
+                                                                 off_val='0'))
+
+        self.add_parameter(name='power',
+                           label='Power',
+                           get_parser=float,
+                           get_cmd='SOUR:POW?',
+                           set_cmd='SOUR:POW {}',
+                           unit='dBm',
+                           vals=Numbers(min_value=-50,
+                                        max_value=10))
+
+        self.add_parameter(name='if_bandwidth',
+                           label='IF Bandwidth',
+                           get_parser=float,
+                           get_cmd='SENS1:BWID?',
+                           set_cmd='SENS1:BWID {}',
+                           unit='Hz',
+                           vals=Enum(1, 3, 1e1, 3e1, 1e2, 3e2,
+                                     1e3, 3e3, 1e4, 3e4, 1e5, 3e5))
+
+        self.add_parameter('averages_enabled',
+                           label='Averages Status',
+                           get_cmd='SENS1:AVER:STAT?',
+                           set_cmd='SENS1:AVER:STAT {}',
+                           val_mapping=create_on_off_val_mapping(on_val='1',
+                                                                 off_val='0'))
+
+        self.add_parameter('averages_trigger_enabled',
+                           label='Trigger average status',
+                           get_cmd='TRIG:SEQ:AVER?',
+                           set_cmd='TRIG:SEQ:AVER {}',
+                           val_mapping=create_on_off_val_mapping(on_val='1',
+                                                                 off_val='0'))
+
+        self.add_parameter('averages',
+                           label='Averages',
+                           get_cmd='SENS1:AVER:COUN?',
+                           set_cmd='SENS1:AVER:COUN {}',
+                           get_parser=int,
+                           set_parser=int,
+                           unit='',
+                           vals=Numbers(min_value=1, max_value=999))
+
+        self.add_parameter(name='start',
```

Comment:
Yes, it is a frequency. But since for a VNA people commonly use start, stop, span and center, I would like to leave it like this.
Furthermore, other vna drivers in qcodes also use this naming scheme, see here for example: https://github.com/QCoDeS/Qcodes/blob/master/qcodes/instrument_drivers/rohde_schwarz/ZNB.py

@FarBo and @jenshnielsen: is there a naming scheme within qcodes for this kind of things? It would make sense to use the same name for the same parameters in all different drivers for the same kind of instruments (e.g. call the start frequency of all vnas start). This would allow to easily change between different physical instruments that do the same thing.

Reply:
I don't think we have strong traditions. I agree that if it is obvious from the context (this being an vna)  (and unit) what this refers to it is fine to stick to short names

Reply:
The name of parameters should only reflect the behavior. Otherwise I think we do not have a naming scheme.
It also makes sense that similar instruments have similar parameter names for the same behavior. 
It is fine to leave the names as they are and I see that ZNB also using this naming scheme, which better to stick to that.